2016-11-17-mvcc-in-tikv.md

```diff
@@ -10,23 +10,23 @@ categories: ['Engineering']
 
 ## Introduction to concurrency control
 
-Transaction isolation is important for database management system. Because database should provide an illusion that the user is the only one who connects to the database, which greatly simplifies application development. But, the concurrency controlling problems like data races must be resolved since there will be a lot of connections to the database. Due to this background, the database management system (DBMS) ensures that the resulting concurrent access patterns are safe, ideally by serializablity.
+In a database system, a transaction might consist of one or more data-manipulation statements and queries, each reading and/or writing information in the database. Users of database systems consider consistency and integrity of data as highly important. However, concurrency controlling problems like data races must be resolved since there will be a lot of connections to the database. Owing to this background, the DBMS must ensure that the resulting concurrent access patterns are safe and ideally serializable.
```

Comment:
I think this paragraph mismatches the technical depth of the rest of the article. Can we maybe shorten it to something close to the last sentence, and say "in this post I will explore the complexities of how to implement such functionality"?

Reply:
Agreed.

Reply:
@morgo I removed this paragraph entirely and add the notion of serializability to the beginning of next paragraph.

Reply:
LGTM